test/Transforms/Passes/tuple-to-struct.mlir

```diff
@@ -0,0 +1,84 @@
+// RUN: p4mlir-opt --p4hir-tuple-to-struct %s | FileCheck %s
+
+!b32i = !p4hir.bit<32>
+!i32i = !p4hir.int<32>
+
+// CHECK-LABEL: module
+// CHECK-LABEL: p4hir.func @bid
+// CHECK: p4hir.return
+// CHECK-NOT: p4hir.tuple
+// CHECK-NOT: p4hir.tuple_extract
```

Comment:
Looks like a copy-paste? There is neither tuple op / tuple_extract ops below. Please ensure that checks are correct.